boards/atmega256rfr2-xpro/include/board.h

```diff
@@ -46,8 +46,8 @@ extern "C" {
  * @name    ztimer configuration values
  * @{
  */
-#define CONFIG_ZTIMER_USEC_ADJUST_SET     (128)
-#define CONFIG_ZTIMER_USEC_ADJUST_SLEEP   (128)
+#define CONFIG_ZTIMER_USEC_ADJUST_SET     128
```

Comment:
Any reasone for removing the `()` here and below?

Reply:
Yes, using IS_ACTIVE in ztimer_init

Reply:
There are all in d87bf1a0b9755889a438f492e6085133cc525fbb

Reply:
Ah, IS_ACTIVE is used on non-(["0","1"]]) defines? is that a good idea?

Reply:
Anything non 0 is ok no?

Reply:
YOu are right, anything non 1 is consideres as inactive..